django/contrib/admin/static/admin/js/core.js

```diff
@@ -1,9 +1,5 @@
 // Core javascript helper functions
 
-// basic browser identification & version
-var isOpera = (navigator.userAgent.indexOf("Opera") >= 0) && parseFloat(navigator.appVersion);
```

Comment:
Opera is still supported.

Reply:
But hasn't Opera long been Chromium-based? Is this even still correct?

Reply:
Yeah, [since 2013](https://en.wikipedia.org/wiki/Opera_(web_browser)#History), Opera has been based on the Chromium blink engine.